app/pages/terms/terms.js

```diff
@@ -13,281 +13,173 @@
-      onBehalf = (
-        <label htmlFor='agreedOnBehalf'>
-          <input
-            id='agreedOnBehalf'
-            type='checkbox'
-            className='js-terms-checkbox'
-            checked={this.state.agreedOnBehalf}
-            onChange={this.handleOnBehalfAgreementChange.bind(this)} />
-          {this.props.messages.ACCEPT_ON_BEHALF}
-        </label>
-      );
-    }
-
-    var acceptTerms;
-    if (this.state.ageSelected !== this.props.ages.NOT_OF_AGE.value) {
-      acceptTerms = (
-        <label htmlFor='agreed'>
+      <p id="checkbox-agreed-terms">
+        <label id="label-checkbox-agreed-terms" htmlFor='input-agreed-terms'>
           <input
-            id='agreed'
+            id='input-agreed-terms'
             type='checkbox'
             className='js-terms-checkbox'
-            checked={this.state.agreed}
-            onChange={this.handleAgreementChange.bind(this)} />
-          {this.props.messages.ACCEPT_OF_AGE}
+            checked={agreed}
+            onChange={() => this.setState({ agreed: !this.state.agreed })} />
+          &nbsp;
+          <span>
+            {t('I am 18 or older and I accept the terms of the')}
+            &nbsp;
+            <a href={urlTermsOfUse} target='_blank' rel='noreferrer'>{textTermsOfUse}</a>
+            &nbsp;
+            {t('and the')}
+            &nbsp;
+            <a href={urlPrivacyPolicy} target='_blank' rel='noreferrer'>{textPrivacyPolicy}</a>
+          </span>
         </label>
-      );
-    }
-
-    var sorryMessage;
-    if (this.state.ageSelected === this.props.ages.NOT_OF_AGE.value) {
-      sorryMessage = (
-        <div ref='sorryMsg'>
-          <p className='terms-sorry-message'>{this.props.messages.SORRY_NOT_OF_AGE}</p>
-        </div>
-    );
-    }
-
-    return (
-      <div className='terms-accept-checkbox'>
-        {acceptTerms}
-        {onBehalf}
-        {sorryMessage}
-      </div>
+      </p>
     );
   }
 
   render() {
-    var content = '';
-    if (this.props.authenticated && !this.props.acceptedLatestTerms) {
-      content = this.renderForm();
-    }
+    const { agreed } = this.state;
+
+    const termsTitle = this.renderTermsTitle();
+    const checkboxAgreedTerms = this.renderCheckboxAgreedTerms(agreed);
 
     return (
-      <div className='terms js-terms'>
-        <LoginNav hideLinks={true} trackMetric={this.props.trackMetric} />
-        <div className='terms-content terms-box'>
-          {content}
+      <div className="container-terms">
+        {termsTitle}
+        {checkboxAgreedTerms}
+        <div id="button-agreed-terms">
+          <button
+              className="btn btn-primary"
+              onClick={this.handleTermsAndPrivacySubmit.bind(this)}
+              disabled={!agreed}>
+            {t('Continue')}
+          </button>
+          <button
+              className="btn btn-secondary"
+              onClick={this.handleRefuseTerms.bind(this)}>
+            {t('Refuse')}
```

Comment:
I would tend to say Reject or Decline
```suggestion
            {t('Reject')}
```